packages/react-button/src/components/Button/__snapshots__/Button.test.tsx.snap

```diff
@@ -97,6 +101,7 @@ exports[`Button renders a default state 1`] = `
     className=
 
         {
+          flex-grow: 1;
```

Comment:
This seems wrong from a design perspective, maybe check with designers. We did not have a fluid mode for v7, so this is new.

Reply:
confirmed what we had is what we want. reverted the change.